_posts/minecraft/bedrock/2024-07-15-server-properties.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "Configuring Bedrock Minecraft Server: Key Attributes of server.properties"
+category: 
+    - Bedrock
+    - Getting-started
```

Comment:
The post only shows up in the Getting-started category, you should only add one category